### PR TITLE
Require a single space after the colon in media features

### PR DESCRIPTION
**Pull request by neveena**
## Suggested rule/changes(s):
* `<media-feature-colon-space-after>` : `severity: ERROR`

@netsells/frontend - Please review

### CODE REVIEW

File: rules/core/media-feature-colon-space-after/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#### 📍 media-feature-colon-space-after
+
+Require a single space after the colon in media features.
+
+##### 🧟 Example of incorrect code for this rule:
+
+```js
```
Comment (samturrell) [inline rules/core/media-feature-colon-space-after/README.md]: Code block should be css and not js :)

**Comment by rowbi-wan** [comment]
🦄

**Review by samturrell — DISMISSED** [review]

**Comment by neveena** [comment]
@samturrell made the changes :)

File: rules/core/media-feature-colon-space-after/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#### 📍 media-feature-colon-space-after
+
+Require a single space after the colon in media features.
+
+##### 🧟 Example of incorrect code for this rule:
```
Comment (rowbi-wan) [inline rules/core/media-feature-colon-space-after/README.md]: Please change to new file structure. see https://github.com/netsells/stylelint-config-netsells/pull/17 for an example

**Review by rowbi-wan — CHANGES_REQUESTED** [review]

**Comment by neveena** [comment]
@rowancollins89 updated the file structure

**Comment by rowbi-wan** [comment]
Need to generate the readme

**Comment by neveena** [comment]
@rowancollins89 updated to new format :)

**Comment by neveena** [comment]
@rowancollins89 made the changes :)

**Review by rowbi-wan — APPROVED** [review]

**Pull request merged**